pkg/terra/client/client.go

```diff
@@ -59,8 +59,13 @@ type Writer interface {
 var _ ReaderWriter = (*Client)(nil)
 
 const (
-	DefaultTimeout            = 10 * time.Second
-	DefaultGasLimitMultiplier = 1.5
+	// DefaultTimeout is the default Terra client timeout.
+	// Note that while the terra node is processing a heavy block,
+	// requests can be delayed significantly (https://github.com/tendermint/tendermint/issues/6899),
+	// however there's nothing we can do but wait until the block is processed.
+	// So we set a fairly high timeout here.
+	DefaultTimeout            = 30 * time.Second
+	DefaultGasLimitMultiplier = 1.2
```

Comment:
The test is failing because this value is too low now "gasWanted: 62049, gasUsed: 62118: out".